### DISCUSSION TITLE
Documentation update for highlighting current path route

### DISCUSSION THREAD

**Discussion by janzheng** [sveltejs/kit#8408] [General]
I've been using Sveltekit (and Sapper) for a really long time, but I've been tripped up trying to figure out how to highlight based on the current pathname. The documentation seems to be missing this info, but this is common enough of a use case I think it should be covered in the documentation (e.g. page.url.pathname)

**Comment by teemingc** [comment]
Currently, there's a nice example in the `<Header>` component of the SvelteKit demo app, available when using `npm create svelte@latest my-app` or navigating to [sveltekit.new](sveltekit.new)

https://stackblitz.com/edit/sveltejs-kit-template-default-rrj5ms?file=src/routes/Header.svelte
```svelte
<ul>
  <li aria-current={$page.url.pathname === '/' ? 'page' : undefined}>
      <a href="/">Home</a>
  </li>
</ul>

<style>
  li[aria-current='page'] {
    /* your style here */
  }
</style>
```
